cogsmisc/customization.py

```diff
@@ -369,6 +370,58 @@ def can_edit_servaliases(ctx):
         parsed = await char.parse_cvars(teststr, ctx)
         parsed = clean_content(parsed, ctx)
         await ctx.send(f"{ctx.author.display_name}: {parsed}")
+                       
+    @commands.command()
+    async def tembed(self, ctx, *, teststr):
```

Comment:
Instead of rewriting the embed code, what you can do is have it parse the input string as above (L369-370), then call the `!embed` command with the parsed string (See sheetManager L179 for an example on how to do this)

Reply:
I think I did it properly? I also removed the imported I added